### ISSUE TITLE
makefiles

### ISSUE THREAD

**Issue by palkinev** [DelNov/T-Flows#17]
write small bash script to extract "use "statements out out .f90 files and turn them into dependencies for make files

**Comment by Niceno** [comment]
This script is of crucial importance, but only for us developers.  Users shouldn't be expected to understand this advance issue, or mess with it in any way (many users will not even know what make command is).

Hence, I suggest we put these script in the "Utilities" directory.

**Comment by palkinev** [comment]
Closed by pull request #24. 

Though, I suppose it can be slightly improved :

1. insert `| sed   -e 's%_Mod_.*.o%_Mod.o%g'  \` [here](https://github.com/DelNov/T-Flows/blob/13cdf1190850eb027129b5c16017f9751ee8c787/Sources/Utilities/script_to_create_external_dependencies_for_makefile.sh#L71)

2. insert 
```
  # Cgns_Void/Seq/Par -> CGNS_OBJ (no such thing)
  # Comm_Seq/Par      -> COMM_OBJ
  # User_Vtu/Cgns      -> ?
  cd $1;
  sed -ie 's%\$(DIR_OBJECT)/Comm_Mod.o%$(DIR_OBJECT)/$(COMM_OBJ)%g' $tmp_file
```

[here](https://github.com/DelNov/T-Flows/blob/13cdf1190850eb027129b5c16017f9751ee8c787/Sources/Utilities/script_to_create_external_dependencies_for_makefile.sh#L250)
